action.yml

```diff
@@ -0,0 +1,186 @@
+name: "End of Life and Outdated JS Scan"
```

Comment:
This is how the comment generated will look
<img width="778" alt="image" src="https://github.com/user-attachments/assets/248cda5f-e263-405a-8aea-e844d57a8cf3" />